packages/quantic/cypress/integration/example-search.cypress.ts

```diff
@@ -25,7 +25,7 @@ describe('example-search', () => {
   });
 
   beforeEach(() => {
-    cy.visit(pageUrl).then(setupAliases).wait('@search');
+    cy.visit(pageUrl).then(setupAliases).wait('@search', {timeout: 30000});
```

Comment:
What's the reason we don't use the `InterceptAliases` here as well?

Reply:
You're right, I forgot to update this test file to use `InterceptAliases.Search`.